extract.go

```diff
@@ -82,6 +82,11 @@ func NewExtractor(filename string, tool string, chooser Chooser) Extractor {
 			File:       chooser,
 			Decompress: gunzipper,
 		}
+	case strings.HasSuffix(filename, ".tgz"):
```

Comment:
Let's compress this into a single line with the previous switch case (join the two case blocks that both use gunzipper):

```go
case strings.HasSuffix(filename, ".tar.gz"), strings.HasSuffix(filename, ".tgz"):
// ...
```